mltsp/Flask/flask_app.py

```diff
@@ -764,19 +768,19 @@ def list_models(
         for this_projkey in authed_proj_keys:
             cursor = (
                 rdb.table("models").filter({"projkey": this_projkey})
-                .pluck("name", "created", "type", "meta_feats").run(g.rdb_conn))
+                .pluck("name", "featureset_name", "created", "type",
+                       "meta_feats").run(g.rdb_conn))
             for entry in cursor:
                 authed_models.append(
                     entry['name']
                     + (" - %s" % str(entry['type']) if with_type else "")
-                    + (
-                        " (created %s PST)" % str(entry['created'])[:-13]
-                        if not name_only else "")
-                    + (
-                        " meta_feats=%s" % ",".join(entry['meta_feats'])
-                        if 'meta_feats' in entry and entry['meta_feats']
-                        not in [False, [], "False", None, "None"]
-                        and isinstance(entry['meta_feats'], list) else ""))
+                    + " (" + entry['featureset_name'] + ") "
```

Comment:
This is difficult to read.  Can we reformat using the newer syntax of:

``"This is my string {0} and {1} and {2}".format(some, args, here)``

or

```
In [8]: "This is the {foo} and the {bar}".format(foo='meep', bar='blep')
Out[8]: 'This is the meep and the blep'
```

(The last one can also be:

```
"This is the {foo} and the {bar}".format(**some_dict)
```